app/modules/user/scss/auth_menu_commons.scss

```diff
@@ -1,29 +1,63 @@
 @import '#general/scss/utils';
 
-.authMenu{
-  label{
-    text-transform: uppercase;
+.auth-menu{
+  /*Small screens*/
+  @media screen and (max-width: $smaller-screen) {
+    .custom-cell{
+      padding: 1em;
+      text-align: center;
+    }
+  }
+
+  /*Large screens*/
+  @media screen and (min-width: $smaller-screen) {
+    @include central-column(28em);
+    min-height: 80vh;
```

Comment:
Since small screens are often used with one hand. It would be nicer for the users to not have to reach the top part of their screen.

Current view:
![2023-11-02_08-14](https://github.com/inventaire/inventaire-client/assets/5363918/aec2abca-b93b-4b0a-9fc1-3634a9d3739c)

What about having the same as the following view but for screens smaller than 600px too:
![2023-11-02_08-11](https://github.com/inventaire/inventaire-client/assets/5363918/77995d09-3295-433e-8876-c7a79b5fb426)

Reply:
Addressed in 6f9b201d4